man/adoc/bpftrace.adoc

```diff
@@ -2415,17 +2415,25 @@ i:s:10 {
 Count how often this function is called.
 
 Using `@=count()` is conceptually similar to `@++`.
-The difference is that the `count()` function uses a map type optimized for this (PER_CPU), increasing performance and correctness.
-Multiple writers to a shared global var might lose counts as bpftrace doesn't update them atomically.
-Due to this the map cannot be accessed as a regular integer.
+The difference is that the `count()` function uses a map type optimized for 
+writing (PER_CPU), increasing performance and correctness. However, sync reads 
+can be expensive as bpftrace needs to iterate over all the cpus to collect and 
+sum these values.
+Note: Multiple writers to a shared global var might lose counts as bpftrace 
+doesn't update them atomically.
```

Comment:
```suggestion
Note: In contrast, multiple writers to a shared global var might lose counts as bpftrace 
doesn't update them atomically.
```

Reply:
Should it be "In contrast to `@++`, which uses a global hash map, multiple writers...

Reply:
Maybe add it on as "(eg `@++`)"